src/components/datagrid/data_grid_cell.tsx

```diff
@@ -39,13 +39,13 @@ import {
   EuiDataGridColumn,
   EuiDataGridPopoverContent,
 } from './data_grid_types';
-import { EuiMutationObserver } from '../observer/mutation_observer';
 import { DataGridFocusContext } from './data_grid_context';
 import { EuiFocusTrap } from '../focus_trap';
 import { keys } from '../../services';
 import { EuiDataGridCellButtons } from './data_grid_cell_buttons';
 import { EuiDataGridCellPopover } from './data_grid_cell_popover';
 
+let count = 0;
```

Comment:
This and the related console.log are probably not necessary :)

Reply:
I thought they were a nice touch :)